### ISSUE TITLE
Installation of core on windows server 2012

### ISSUE THREAD

**Issue by anjanadv** [jembi/openhim-core-js#980]
Hello,
We have installed OpenHIM on a Windows Server 2012 VM following the documentation. But we are not able to access the Core API from the console. The error thrown by the Console while trying to login is as follows - 
![image](https://user-images.githubusercontent.com/18394680/44792999-d4faa000-abc2-11e8-8763-a9cf0b3c7bbc.png)

When trying to browse the API on port 8080 it is giving the following error - 
![image](https://user-images.githubusercontent.com/18394680/44793073-05423e80-abc3-11e8-9c30-7a936e2e0875.png)

Both the core and console are running on the same server. Core is setup as a windows service following the installation guidelines. There is no error being logged in stderr.log and stdout.log. 
Console is hosted on port 80 on IIS. And core is also running on the default ports.

How can we resolve this issue?
Thanks.

**Comment by BMartinos** [comment]
Hi @anjanadv The issue with the login seems to be related to the Self Signed certificate that gets created when installing the OpenHIM core for the first time. Clicking the link in the Console error box should present you with a browser error page to explicitly accept the Self Signed certificate. Once its been accepted, the login process should proceed.

I do however see based on your screenshot that you arent presented with the "Advanced" option in Chrome to allow the insecure certificate which is strange. Are you able to telnet to that port to verify its open and allowing request?

**Comment by anjanadv** [comment]
Yes, We are able to telnet to that port. And this issue is not specific to one browser. We have observed this in all the browsers and multiple servers.

**Comment by rcrichton** [comment]
@anjanadv this could rather be an issue with the OpenHIM core not starting up correctly. Do you see any issues in the Core logs?

**Comment by anjanadv** [comment]
@rcrichton , No, there is no error being logged in the files. The core service seems to be running absolutely fine. Attached is the screenshot of stdout, stderr is empty.
![log](https://user-images.githubusercontent.com/18394680/45019591-5d0cf980-b04a-11e8-8721-ea8a2e192c85.png)

During initialization(running for the first time after installing), we noticed that it is skipping 2 certificates(screenshot below), would that be a reason for the error?

![screenshot 1](https://user-images.githubusercontent.com/18394680/45027524-44f3a500-b05f-11e8-81cc-e49a107ee27e.png)

**Comment by anjanadv** [comment]
Yes, we can see the entry 'API HTTPS Listening on port 8080'. 
 We are getting the following when we run then curl
**curl -k https://localhost:8080/heartbeat
curl: (35) error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure**

**Comment by rcrichton** [comment]
Sorry, I haven't seen this before. It could be an issue with self signed certificates on windows. I did find this article. Perhaps it can be of help - https://medium.com/@sslsecurity/how-to-fix-err-ssl-version-or-cipher-mismatch-error-5447c30ac78a
